src/store/actions/SovereignVenueId.ts

```diff
@@ -0,0 +1,32 @@
+export enum SovereignVenueIdActionTypes {
+  SET_SOVEREIGN_VENUE_ID = "SET_SOVEREIGN_VENUE_ID",
+  SET_SOVEREIGN_VENUE_IS_LOADED = "SET_SOVEREIGN_VENUE_IS_LOADED",
+}
+
+type setSovereignVenueIdIsLoadingAction = {
```

Comment:
the `isLoading` naming is here explicitly. I want to reduce the number of calls to the API to only one. And the only way to to it is to have if !`isLoading` set `isLoading` to true. You can't do the same with `isLoaded`. I'm open to suggestions though

Reply:
I'm not quite sure I am following your description of your intended flow with this.

When you refer to API, do you mean the backend firestore API/query/etc? Or are you referring to an 'internal' API with Redux/ec?

---

The change I called out above is because the name of this type is 'Isloading', and yet the `type` field is set to `IS_LOADED`. So at the most basic level (regardless of higher level design patterns/intent), it seems like both of these should be using the same 'term' (either loading, or loaded)

Reply:
> I'm not quite sure I am following your description of your intended flow with this.
> 
> When you refer to API, do you mean the backend firestore API/query/etc? Or are you referring to an 'internal' API with Redux/ec?

I mean the backend firestore


> The change I called out above is because the name of this type is 'Isloading', and yet the type field is set to IS_LOADED. So at the most basic level (regardless of higher level design patterns/intent), it seems like both of these should be using the same 'term' (either loading, or loaded)

I see. I didn't get it from the suggestion.